### PR TITLE
Purview Catalog

### PR DESCRIPTION
**Pull request by qwordy**

### CODE REVIEW

File: sdk/purview/azure-analytics-purview-catalog/README.md [hunk 1]
```diff
@@ -58,10 +58,10 @@ Set the values of the client ID, tenant ID, and client secret of the AAD applica
 ##### Example
```
Comment (weidongxu-microsoft) [inline sdk/purview/azure-analytics-purview-catalog/README.md:58]: [//]: # ({x-version-update-end})

File: sdk/purview/azure-analytics-purview-catalog/pom.xml [hunk 2]
```diff
@@ -40,7 +40,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.20.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.21.0-beta.1</version> <!-- {x-version-update;beta_com.azure:azure-core;dependency} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
```
Comment (weidongxu-microsoft) [inline sdk/purview/azure-analytics-purview-catalog/pom.xml:46]: clean up the test dependency that you do not use.

File: sdk/purview/azure-analytics-purview-catalog/swagger/README.md [hunk 3]
```diff
@@ -1,14 +1,28 @@
...
-sync-methods: none
+sync-methods: all
 license-header: MICROSOFT_MIT_SMALL
 low-level-client: true
 credential-types: tokencredential
 credential-scopes: https://purview.azure.net/.default
 title: PurviewCatalogClient
+service-name: PurviewCatalog
+service-versions:
+  - "2021-09-01"
```
Comment (weidongxu-microsoft) [inline sdk/purview/azure-analytics-purview-catalog/swagger/README.md:16]: Do you need to add the preview version here?

File: sdk/purview/azure-analytics-purview-catalog/README.md (hunk 1 above)
Comment (weidongxu-microsoft) [reply sdk/purview/azure-analytics-purview-catalog/README.md:58]:
```
[//]: # ({x-version-update-end})
```

**Review by weidongxu-microsoft — APPROVED** [review]

File: sdk/purview/azure-analytics-purview-catalog/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2021-09-30)
+- API version 2021-09-01
```
Comment (weidongxu-microsoft) [inline sdk/purview/azure-analytics-purview-catalog/CHANGELOG.md:4]: Need a better changelog.

Comment (weidongxu-microsoft) [reply sdk/purview/azure-analytics-purview-catalog/CHANGELOG.md:4]: https://azure.github.io/azure-sdk/policies_releases.html#change-logs

File: sdk/purview/azure-analytics-purview-catalog/src/samples/java/com/azure/analytics/purview/catalog/GlossaryListGlossaries.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+        GlossaryClient client =
+                new PurviewCatalogClientBuilder()
+                        .endpoint(System.getenv("ENDPOINT"))
+                        .credential(new DefaultAzureCredentialBuilder().build())
+                        .buildGlossaryClient();
+        RequestOptions requestOptions = new RequestOptions();
+        requestOptions.addQueryParam("limit", "1");
+        requestOptions.addQueryParam("sort", "ASC");
+        requestOptions.addQueryParam("offset", "0");
+        Response<BinaryData> response = client.listGlossariesWithResponse(requestOptions, null);
```
Comment (weidongxu-microsoft) [inline sdk/purview/azure-analytics-purview-catalog/src/samples/java/com/azure/analytics/purview/catalog/GlossaryListGlossaries.java:23]: Use `Context.NONE`

File: sdk/purview/azure-analytics-purview-catalog/swagger/README.md (hunk 3 above)
Comment (qwordy) [reply sdk/purview/azure-analytics-purview-catalog/swagger/README.md:17]: Added

File: sdk/purview/azure-analytics-purview-catalog/README.md (hunk 1 above)
Comment (qwordy) [reply sdk/purview/azure-analytics-purview-catalog/README.md:58]: Done

File: sdk/purview/azure-analytics-purview-catalog/pom.xml (hunk 2 above)
Comment (qwordy) [reply sdk/purview/azure-analytics-purview-catalog/pom.xml:46]: Done

File: sdk/purview/azure-analytics-purview-catalog/src/samples/java/com/azure/analytics/purview/catalog/GlossaryListGlossaries.java (hunk 5 above)
Comment (qwordy) [reply sdk/purview/azure-analytics-purview-catalog/src/samples/java/com/azure/analytics/purview/catalog/GlossaryListGlossaries.java:23]:
```suggestion
        Response<BinaryData> response = client.listGlossariesWithResponse(requestOptions, Context.NONE);
```

Comment (qwordy) [reply sdk/purview/azure-analytics-purview-catalog/src/samples/java/com/azure/analytics/purview/catalog/GlossaryListGlossaries.java]: done

File: sdk/purview/azure-analytics-purview-catalog/CHANGELOG.md (hunk 4 above)
Comment (qwordy) [reply sdk/purview/azure-analytics-purview-catalog/CHANGELOG.md]: Added

File: sdk/purview/azure-analytics-purview-catalog/CHANGELOG.md [hunk 6]
```diff
@@ -1,7 +1,15 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2021-09-30)
```
Comment (qwordy) [inline sdk/purview/azure-analytics-purview-catalog/CHANGELOG.md:3]:
```suggestion
## 1.0.0-beta.2 (unreleased)
```

File: sdk/purview/azure-analytics-purview-catalog/CHANGELOG.md [hunk 7]
```diff
@@ -1,7 +1,15 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (unreleased)
+### Features Added
+- Adopt new API version 2021-09-01. It adds some new operations.
 
+### Breaking Changes
+This a new version of LLC SDK. Changes are
```
Comment (qwordy) [inline sdk/purview/azure-analytics-purview-catalog/CHANGELOG.md:8]:
```suggestion
This is a new version of LLC SDK. Changes are
```

**Pull request merged**